### PR TITLE
chore: move to gh actions

### PR DESCRIPTION
**Pull request by frezbo**
Move to using GitHub actions.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -77,7 +77,6 @@ NONFREE_TARGETS = \
 
 all: $(TARGETS) ## Builds all known pkgs.
 
-.PHONY: nonfree
```
Comment (smira) [inline Makefile:80]: not sure why this got removed, if we ever create path `nonfree`, this will stop working as a target

**Review by smira — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (frezbo) [reply Makefile:80]: this was not present in `pkgs` and we run `make` and `make nonfree` separately

Does it make sense to build `nonfree` when just `make` is invoked?

Comment (smira) [reply Makefile:80]: it's not about `all` in fact, but it's about how `make` works. without `.PHONY`, if the file named `nonfree` exists, `make` will skip the target. `.PHONY` says that this target actually doesn't produce a file named `nonfree`.

Comment (frezbo) [reply Makefile:80]: ahh yes, i forgot that

**Pull request merged**